sample.mvn.parent/pom.xml

```diff
@@ -14,10 +16,43 @@
 	</parent>
 
 	<properties>
+		<!-- General Project Properties -->
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+		<!-- External Tools Properties -->
+		<bnd-bnd-maven-plugin-version>5.0.1</bnd-bnd-maven-plugin-version>
+		<m2e-lifecycle-mapping-version>1.0.0</m2e-lifecycle-mapping-version>
+
+		<!-- OSGi Properties -->
+		<osgi-org.osgi.service.component.annotations-version>1.3.0</osgi-org.osgi.service.component.annotations-version>
 		<osgi.qualifier>${maven.build.timestamp}</osgi.qualifier>
+
+		<!-- BestSolution/Efxclipse Properties -->
+		<bestsolution-maven-osgi-exec-plugin-version>0.0.3-SNAPSHOT</bestsolution-maven-osgi-exec-plugin-version>
+		<bestsolution-maven-osgi-package-plugin-version>0.0.3-SNAPSHOT</bestsolution-maven-osgi-package-plugin-version>
+		<efxclipse.rt-org.eclipse.equinox.executable-version>3.8.100-SNAPSHOT</efxclipse.rt-org.eclipse.equinox.executable-version>
+		<org.eclipse.fx.ui.workbench.fx.application-version>1.0.0</org.eclipse.fx.ui.workbench.fx.application-version>
+
+		<!-- OpenJFX Properties -->
+		<openjfx-javafx-controls-version>11</openjfx-javafx-controls-version>
+		<openjfx-openjfx.base-version>11.0.0</openjfx-openjfx.base-version>
```

Comment:
can this ones be united into one or should they be kept separated?

Reply:
I think it is better to just have one openjfx-version attribute

Reply:
sure, if the versioning is always the same in those packages, that's the way to go

Reply:
yes they are all the same for all packages all the time